src/ui/features/projects/ProjectForm/ProjectForm.test.tsx

```diff
@@ -204,6 +225,30 @@ describe('ProjectForm', () => {
 
       expect(list.textContent).toContain('СССР');
     });
+
+    it('очищает поле при редактировании', async () => {
+      const onSubmit = jest.fn().mockImplementation((values) => Promise.resolve(values));
+
+      const initialValues = {
+        ...initializeProjectForm(),
+        region: 'region-1',
+      };
+
+      renderComponent({ initialValues, onSubmit });
+
+      const comboboxElement = screen.getByTestId(DescriptionStep.testId.region);
+
+      const combobox = getCombobox(comboboxElement);
+
+      await combobox.clear();
+
+      await act(async () => {
+        await submitForm();
+      });
+
+      expect(onSubmit).toBeCalled();
+      expect(onSubmit.mock.calls[0][0]).toEqual(expect.objectContaining({ region: null }));
```

Comment:
Должно же и так работать
```suggestion
      expect(onSubmit).toHaveBeenLastCalledWith(expect.objectContaining({ region: null }));
```

пример из доки
```js
// The last call to the mock function was called with the specified args
expect(mockFunc).toHaveBeenLastCalledWith(arg1, arg2);
```